docs/user/alerting/rule-types/es-query.asciidoc

```diff
@@ -60,4 +60,32 @@ image::user/alerting/images/rule-types-es-query-valid.png[Test {es} query return
 * An error message is shown if the query is invalid.
 +
 [role="screenshot"]
-image::user/alerting/images/rule-types-es-query-invalid.png[Test {es} query shows error when invalid]
\ No newline at end of file
+image::user/alerting/images/rule-types-es-query-invalid.png[Test {es} query shows error when invalid]
+
+[float]
+==== Match de-duplication
+
+The {es} query rule type performs de-duplication of document matches across rule executions. This means that when the rule is configured with a smaller schedule interval than time window, a document may match a query in multiple rule executions but will be alerted on at most once.
```

Comment:
```suggestion
The {es} query rule type performs de-duplication of document matches across rule executions. If you configure the rule with a schedule interval smaller than the time window, and a document matches a query in multiple rule executions, it will be alerted on only once.
```